content/posts/unpublished/backbase-tech-stack/post.md

```diff
@@ -0,0 +1,198 @@
+to integrate with different core banking systems.
+
+To achieve this, different technologies, frameworks, standards, and architectural patterns need to be combined.
+
+
+This article aims to provide an insight into how the Backbase software is built and integrated on a high level,
+what kind of technologies are used on a different levels and layers.
+
+## High level architecture
+
+<img align="right" src="assets/HL_Architecture.drawio.png">
+
+Engagement Banking Services are divided according to the capabilities they offer, the role they play, and on which level they operate.
+
+Each service is designed to perform completely different tasks and offers specific ways to interact with it.
+
+On the top there are client-facing apps. These applications are available for 2 different platforms:
+
+* Web applications - available on modern web-browsers, dedicated for different group of users, segregated by the roles and responsibilities
+* Mobile applications - available on popular mobile devices powered by iOS and Android Mobile OSs
+
+These applications allow to interact with backend services for these users who have proper permissions.
+Client APIs are a dedicated and well secured APIs designed for interaction between client-facing applications and backend services.
+
+On the next layer, there are services which take care of the security of the client-facing applications.
+These layer of services is able to recognize the user and introduce the same user to backend services in a more understandable way.
+This layer also offers the set of generic services which delivers generic capabilities like document storage or auditing for all other specific services.
+
+Next layer offers specific banking services like Arrangements, Batches, Cards, Contacts, Payments etc.
+These services offer set of functions oriented on a specific banking capability.
+Interaction between services on this layer is possible through the Service APIs.
+
+To interact with Core Banking services or Third Party services, there is a dedicated integration layer.
+Banking Services have defined set of integration APIs for outbound and inbound connectivity.
+Outbound APIs are for communication from BB to Core. Inbound APIs are from communication from Core to BB.
+Thanks to this asynchronous data exchange is easy achievable and the load can be optimised by the Core.
+In many scenarios, Integration Services are implemented as a separate layer where the communication with Core is defined
+and custom logic is implemented.
+This approach gives the flexibility of the solution because there is one shared API spec between Banking and Integration Services,
+and what is more Integration Services can deliver something specific which is required to properly deliver integration with Core Banking.
+
+Backbase has also a new way for integrating with Core and Third Parties. It was introduced in 2024, and it is called Grand Central.
```

Comment:
if you are mentioning grand central you also have to cover all of its tech stack which is different from what we have here

Reply:
I added more info about GC and what kind of integration it provides

Reply:
and where did you put the GC stack? camelk, jolt, xlst, etc

https://backbase.io/documentation/gc-connectors/2024.12/reference/open-source-standards-technologies

Reply:
I have already added dedicated subsection for GC stack